contents/blog/posthog-vs-fathom.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: "In-depth: PostHog vs Fathom"
+date: 2024-11-11
+author:
+ - bijan-boustani
+rootpage: /blog
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_vs_fathom_analytics_e327b58767.jpg
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonHeader } from 'components/ComparisonTable/header'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+PostHog and Fathom Analtyics both provide alternatives to Google Analytics. In fact, both explicitly mention it with marketing copy like "Switch to a better Google Analytics alternative."
+
+Both products feature web analytics dashboards with an emphasis on simplicity and privacy. They're ideal choices for customers that want to analyze and understand web traffic but find Google Analytics to be needlessly complicated.
+
+But what makes them different?
+
+## How is PostHog different?
+
+### 1. PostHog offers web analytics, and a lot more
+
+[Web analytics](/web-analytics) is a core feature of PostHog, but we offer many more tools to help you analyze, test, and observe your site and your product.
+
+These include [product analytics](/product-analytics), [session replay](/session-replay), [surveys](/surveys), [experiments](/experiments), [feature flags](/feature-flags), a [data warehouse](/data-warehouse), and more.
+
+### 2. PostHog has a generous free tier
+
+PostHog's ["ridiculously cheap" tier](/pricing) includes all of the features of PostHog for free with an allowance of 1M events per month. For Fathom, the pricing for up to 1M data points is $60 per month or 100K data points for $15 per month.
+
+Beyond PostHog's free tier, we try to be as cheap as possible and [transparent](/docs/billing/estimating-usage-costs). Want to know how much we'll charge? See our [pricing calculator](/pricing#calculator).
+
+PostHog also does not require a credit card to sign up and try the free tier.
+
+### Is Fathom simpler than PostHog?
```

Comment:
I see what you're trying to do here, but this doesn't really go under the section: "How is PostHog different?" 

If someone didn't know much about either (or just knew Fathom), they wouldn't immediately assume PostHog was more complicated.

Reply:
Yeah that's a good point.

This was originally under the FAQ at the bottom, but I was thinking readers would benefit from seeing screenshots somewhere earlier in the post (especially since the similarities are pretty obvious when we look at the dashboards).

But you're right it doesn't fit too well here. I'll take a look at a change now. I wonder if it's partly just this heading that seems out of place.